app/Events/Listeners/CheckAttachmentListener.php

```diff
@@ -13,6 +13,10 @@ public function handle(CheckAttachment $event)
                 if ($attachment) {
                     $event->object->attachments()->create([
                         'user_id' => auth()->user()->id,
+                        /*

```

Comment:
Comment can be removed

Reply:
Resolved in commit 2b00f09: added comment in commit message and removed it from the source